reportportal/templates/api-deployment.yaml

```diff
@@ -111,6 +111,18 @@ spec:
         - name: RP_BINARYSTORE_MINIO_SECRETKEY
           value: "{{ .Values.minio.secretkey }}"
        {{ end }}
+       {{ if .Values.minio.bucketPrefix }}
+        - name: RP_BINARYSTORE_MINIO_BUCKETPREFIX
+          value: "{{ .Values.minio.bucketPrefix }}"
+       {{ end }}
+       {{ if .Values.minio.defaultBucketName }}
+        - name: RP_BINARYSTORE_MINIO_DEFAULTBUCKETNAME
+          value: "{{ .Values.minio.defaultBucketName }}"
+       {{ end }}
+       {{ if .Values.minio.defaultBucketName }}
```

Comment:
The RP_INTEGRATION_SALT_PATH is another bucket name which different to the default bucket name, default without config it bucket name could be `keystore`.

With bucket prefix set, the integration salt path name will combined with prefix.

So here could create a new parameter `integrationSaltPath` under minio parameters, then check whether this value is set.

Reply:
Ok, i think i've set this up as requested.